lib/matplotlib/_mathtext.py

```diff
@@ -2527,7 +2527,7 @@ def subsuper(self, s: str, loc: int, toks: ParseResults) -> T.Any:
                         hasattr(new_children[-2], '_metrics')):
                     new_children = new_children[:-1]
                 last_char = new_children[-1]
-                if hasattr(last_char, '_metrics'):
+                if isinstance(last_char, Char):
```

Comment:
Should we also change the `hasattr` 3 lines up?

Reply:
Sure!